package.nix

```diff
@@ -166,6 +166,11 @@ goBuild {
     ++ lib.optionals enableCuda cudaLibs
     ++ lib.optionals stdenv.hostPlatform.isDarwin metalFrameworks;
 
+  patches = [
+    # sin-eating: ollama's build script is unable to find hipcc
```

Comment:
I'm not familiar with the term "sin-eating". Does it have any technical meaning? I looked at the [sin-eater wikipedia article](https://en.wikipedia.org/wiki/Sin-eater), but I feel like I still don't get what you meant.

Reply:
it was a reference to nixcon 2024 where one of the presenters referred to nixpkgs as sin-eaters of linux app packaging.